tests/components/zwave_js/conftest.py

```diff
@@ -12,27 +12,33 @@
 from zwave_js_server.model.node import Node
 from zwave_js_server.version import VersionInfo
 
+from homeassistant.components.zwave_js.const import DOMAIN
 from homeassistant.core import HomeAssistant
+from homeassistant.util.json import JsonArrayType
 
-from tests.common import MockConfigEntry, load_fixture
+from tests.common import (
+    MockConfigEntry,
+    load_json_array_fixture,
+    load_json_object_fixture,
+)
 
 # State fixtures
 
 
 @pytest.fixture(name="controller_state", scope="package")
```

Comment:
In the future we can probably drop the package scope, since `load_fixture` already caches the return values. Then we can also remove the `deepcopy` of the data when creating the nodes.

Reply:
Oh really? I've been told multiple times that I need to separate the fixture and add a scope to it to make sure we don't have to load every time

Reply:
Yeah, that's what I normally do. But since we have a cache on the function that reads from disk it may not be needed. Would be good to do some performance testing though to check what is faster.